docs/notebooks/Common_Gotchas_in_JAX.md

```diff
@@ -1113,6 +1113,12 @@ Note that #2-#4 work for _any_ of JAX's configuration options.
 
 We can then confirm that `x64` mode is enabled:
 
+```{code-cell} ipython3
+:tags: [remove-cell]
+
+jax.config.update("jax_enable_x64", True)
```

Comment:
I don't think we want to change this flag value in the notebook. What if we put the `print x.dtype` in a markdown block rather than in a code-cell, so we don't actually execute it?

Reply:
Much better suggestion! I should have thought of that. Thanks @jakevdp! Updated accordingly.